docs/developer-guide/actions/writing-playbooks.rst

```diff
@@ -136,6 +136,52 @@ Sometimes you need to prevent an action from running too often. You can use the
 
 The second parameter to ``RateLimiter.mark_and_test`` defines a key used for checking the rate limit. Each key is rate-limited individually.
 
+Flow control
+-------------
+Every Robusta playbook has a list of :ref:`Triggers` and a list of :ref:`Actions`.
+
+.. code-block:: yaml
+
+   - triggers:
+     - on_deployment_create: {}
+     - on_daemonset_update: {}
+     actions:
+     - resource_babysitter: {}
+     - add_deployment_lines_to_grafana: {}
+
+For every incoming event, Robusta will run any playbook, that *one* of it's ``triggers`` is matched.
+
+All the playbook's ``actions`` will be executed, according to the order specified in the configuration.
+
+Playbooks execution order is according to the order specified in the configuration as well.
```

Comment:
This sentence "Playbooks execution order is according to the order specified in the configuration as well." seems repetetive and to repeat the information in the previous paragraph.

I know what you're trying to say and I don't think it's getting across. Not sure how to better phrase it. Maybe just give an example and show not tell.

Reply:
Changed the description to: Playbooks execution order, is the same as the configuration file order.
added an example as well